webapp/channels/src/components/user_account_menu/user_account_away_menuitem.tsx

```diff
@@ -73,7 +73,6 @@ export default function UserAccountAwayMenuItem(props: Props) {
                 />
             }
             trailingElements={trailingElement}
-            aria-checked={props.isStatusAway}
```

Comment:
Axe is complaining about these because the `menuitem` role on these items doesn't support `aria-checked`. Instead of removing the `aria-checked` though, we should change the role of these to `menuitemradio`.

We might also consider making a `Menu.RadioItem` which automatically sets that role, the `aria-checked`, and the `trailingElement`, but that's probably outside the scope of this PR since there might be more to consider for that like how to group the items